fa20/syllabus.yml

```diff
@@ -21,12 +21,12 @@
     authors: []
     filename: meeting01
     cover-image: ''
-    tags: []
+    tags: ["ai", "machine learning"]
```

Comment:
These are somewhat generic tags. A good start, but maybe add more specific things that describe what the paper gets at? (e.g. keywords about how the models enhance the video games.)